test/TestUtilities/Test.Utility/MockServer.cs

```diff
@@ -104,15 +104,13 @@ public void Stop()
         {
             try
             {
-                _listener.Abort();
+                _listener?.Abort();
```

Comment:
This logic was causing a `NullReferenceException` sometimes and would crash tests.  This also might make things more reliable...